src/main/resources/templates/search.ftl

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h3>Search page</h3>
+<a href="http://localhost:8080/">Home</a>
+<a href="http://localhost:8080/add/">Add blog</a>
+<fieldset>
+    <legend>Search a blog</legend>
+    <form>
+        Text for search: <input type="text" name="text"/> <br/>
+        <input type="submit" value="Search" />
+    </form>
+</fieldset>
+<#if blogs?has_content>
+    <ul>
+        <#list blogs as blog>
+        <li>
```

Comment:
displaying of list item can be put in a separated template with a name like "blogItem.ftl", probably it would be even better to put "ul" tag in this template as well